doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2996,36 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: A primary purpose of most Dataverse installations is to serve an archive. In the archival space, there are best practices around the tracking of data access and the tracking of modifications to data and metadata. In support of these key workflows, a simple mechanism to delete users that have performed edit or access actions in the system is not provided. Providing a Disable User endpoint instead of a Delete User endpoint is by design.
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all Dataverse collections, datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
```

Comment:
We could add:

- Prevent a user from being assigned roles.

Reply:
added in https://github.com/IQSS/dataverse/commit/c85691bc19fc88b5f1df2d46b63d121bfa1dc16e